rules/hiredis.mk

```diff
@@ -1,7 +1,7 @@
 # libhiredis package
 
 HIREDIS_VERSION = 0.14.0
```

Comment:
The version 0.14-0 should be a version introduced in the old Debian release. For bullseye, can we use version 0.14.1-1?
```
root@b0889ce3a4c4:/# cat /etc/os-release | grep VERSION_CODENAME
VERSION_CODENAME=bullseye
root@b0889ce3a4c4:/# apt-cache madison libhiredis0.14
libhiredis0.14 |   0.14.1-1 | http://deb.debian.org/debian bullseye/main amd64 Packages
root@b0889ce3a4c4:/# 

```

Reply:
@xumia fixed